src/Form/WmContentContainerForm.php

```diff
@@ -133,18 +124,9 @@ public function form(array $form, FormStateInterface $form_state)
             '#description' => t('Allowed bundles in this type.'),
         ];
 
-        $child_bundles = [];
-        if ($entity->getHostEntityType()) {
-            $child_bundles = $this->getAllBundles($entity->getChildEntityType());
-        } elseif (isset($values['child_entity_type'])) {
```

Comment:
This was all being done because we need to provide all bundles when no check boxes are done. Since this is now taken out of the form instead put into entity we don't need this ugly build up any more.